Sources/Swift/Tools/LoadValidator.swift

```diff
@@ -0,0 +1,90 @@
+@_implementationOnly import _SentryPrivate
+import Darwin
+import Foundation
+import MachO
+
+@objc
+@_spi(Private) public final class LoadValidator: NSObject {
+    // Any class should be fine
+    static let targetClassName = "PrivateSentrySDKOnly"
+    
+    @objc
+    @_spi(Private) public class func validateSDKPresenceIn(_ image: SentryBinaryImageInfo) {
```

Comment:
`m`: I'm a bit confused by the method name. This method and this class validate that the SDK isn't loaded twice. Does it really validate the SDK presence? It prints a warning when it finds the SentrySDK loaded in two binaries.

Reply:
Renamed to `checkForDuplicatedSDK`